Readme.md

```diff
@@ -4,14 +4,20 @@
 As the installer is a part of the Clear Linux OS distribution, this program follows the [Clear Linux OS Security processes](https://clearlinux.org/documentation/clear-linux/concepts/security).
 
 ## Dependencies
-The following bundles are required in order to run clr-installer:
+The to ensure you have the required bundles in order to build and run clr-installer, install the latest clr-installer bundle:
```

Comment:
Remove the starting "The to ensure.."

Side note: I would avoid the usage of "you" in technical documentation. Your choice :)
E.g. I would just have the sentence as 
**"In order to build and run clr-installer, install the latest clr-installer bundle:"**